tests/unit/amo/pages/TestSearchTools.js

```diff
@@ -55,33 +53,4 @@ describe(__filename, () => {
 
     expect(root.find(HeadLinks)).toHaveLength(1);
   });
-
-  describe('enableFeatureRemoveSearchTools = true', () => {
-    it('sends a server redirect', () => {
-      const _config = getFakeConfig({ enableFeatureRemoveSearchTools: true });
```

Comment:
Will it be as it is now ? or what should we set to `_config` here ?

Reply:
no need to pass a `_config` object, it is by default now.